tests/example.spec.ts

```diff
@@ -1,21 +1,16 @@
 import { test, expect } from '@playwright/test';
 
-test('has title - visual comparison', async ({ page }) => {
-  await page.goto('https://getbootstrap.com');
+test('Visual regression - Masking', async ({ page }) => {
+  await page.goto('https://www.netflix.com');
 
   // Expect a title "to contain" a substring.
-  await expect(page).toHaveTitle(/Bootstrap/);
-});
-
-test('get started link - visual comparison', async ({ page }) => {
-  await page.goto('https://playwright.dev/');
-
-  // Click the get started link.
-  await page.getByRole('link', { name: 'Get started' }).click();
-
-  // Expects the URL to contain intro.
-  await expect(page).toHaveURL(/.*intro/);
+  await expect(page).toHaveTitle(/Netflix/);
 
   // Perform visual comparison
-  await expect(page).toHaveScreenshot()
-});
+  await expect(page).toHaveScreenshot({
+    fullPage: true,
+    mask: [page.locator('#appMountPoint > div > div > div > div > div > div > div.our-story-cards > div:nth-child(3) > div.default-ltr-cache-b2rltc.ejqbulh1 > div.default-ltr-cache-7cljuy.ejqbulh3 > div'), 
+    page.locator('#appMountPoint > div > div > div > div > div > div > div.our-story-cards > div:nth-child(4) > div.default-ltr-cache-51sor6.ejqbulh1 > div.default-ltr-cache-vgp0nn.ejqbulh3 > div'), 
+    page.locator('#appMountPoint > div > div > div > div > div > div > div.our-story-cards > div:nth-child(6) > div.default-ltr-cache-51sor6.ejqbulh1 > div.default-ltr-cache-vgp0nn.ejqbulh3 > div > div > div.default-ltr-cache-wmj6sw.ecjxv8n5')]
```

Comment:
Is there a way to simplify these selectors? Any additional elements introduced between these chains of selectors would break it.

Reply:
I've refactored and simplified these selectors. I'm masking the elements by role, with less code now. Thanks for that heads up!